src/components/IconTooltip.tsx

```diff
@@ -0,0 +1,56 @@
+import { Theme, Tooltip, TooltipProps } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+import Icon from './Icon/Icon';
+import { IconName } from './Icon/icons';
+import { useDeviceInfo } from '../utils';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  arrow: {
+    color: '#3A4445',
+    marginLeft: '3px',
+    marginTop: '-1px',
+  },
+  icon: {
+    fill: '#3A4445',
+    verticalAlign: 'text-top',
+  },
+  tooltip: {
+    backgroundColor: '#3A4445',
+    borderRadius: '8px',
+    padding: '8px',
+    '& a': {
+      color: '#fff',
+    },
+  },
+}));
+
+type IconTooltipProps = {
+  iconName?: IconName;
+  placement?: TooltipProps['placement'];
+  title: TooltipProps['title'];
+};
+
+export default function IconTooltip({ iconName = 'info', placement = 'top-start', title }: IconTooltipProps): JSX.Element {
+  const { isMobile } = useDeviceInfo();
+  const classes = useStyles();
+
+  return (
+    <Tooltip
```

Comment:
Curious whether `Tooltip` is powerful enough for our use case. As far as I can tell we can't add react components into the `title` prop, which might make adding a dialog trigger for "Learn More" difficult. An alternative might be [`Popover`](https://mui.com/material-ui/react-popover/). I may just not know this component well enough, though. Thoughts?

Reply:
Apparently we can add html into the title, so we could add a component, correct?

Reply:
@charlie-tf yeah the type of `title` is `NonNullable<React.ReactNode>`, same as the `title` prop on the `Tooltip` component from MUI.

Reply:
Awesome, thanks!